.github/workflows/script

```diff
@@ -1,12 +1,9 @@
 #!/bin/bash
 set -ex
 
-git remote set-branches --add origin $TRAVIS_BRANCH
-git fetch
-
-if [[ ( "$TRAVIS_PULL_REQUEST" != "false" ) && ( "$TEST_SUITE" == "lint" ) ]]; then
+if [[ -n "$GITHUB_BASE_REF" ]] && [[ "$TEST_SUITE" == "lint" ]]; then
```

Comment:
I was not able to test this bit yet, because `GITHUB_BASE_REF` is only set in pull requests according to the documentation.